website/routes.py

```diff
@@ -175,14 +175,19 @@ def make_url_map(app):
 
     process_rules(app, [
 
-        Rule('/dashboard/', 'get', website_views.dashboard, OsfWebRenderer('dashboard.mako')),
+        Rule('/dashboard/', 'get', website_views.redirect_to_myprojects, OsfWebRenderer('dashboard.mako')),
```

Comment:
Dashboard is not anymore the dashboard though. The place it goes to is not dashboard, it's a page called My Projects. The meaning of dashboard is not present in current page. Do we have to hold on to the name?